README.md

```diff
@@ -35,6 +35,29 @@ The Wukong CLI is a set of tools to manages Mindvalley DevOps resources. Its goa
 
 * Currently the Jenkins sometimes does not return the list of changes during build, so the CLI is not able to determine the CHANGELOG properly. We are working on a solution in the future release.  
 
+## IMPORTANT !!
+
+The latest release of Wukong CLI contains several breaking changes. User must upgrade to the latest version, which is at least `0.0.4-beta1` in order to continue using the Wukong CLI. Please follow the below instructions:
+
+```
+# Check the current version of Wukong CLI.  
+$ wukong --version
+
+# If your version is < 0.0.4-beta1.  
+$ brew update  
+$ brew upgrade wukong  
+
+# Confirm that you're on version >= 0.0.4-beta1.  
+$ wukong --version
+wukong 0.0.4-beta1
+
+# Delete the current config.  
+$ rm ~/.config/wukong/config.toml
+
+# Re-init the Wukong CLI.  
+$ wukong init
+```
+
```

Comment:
```suggestion
## IMPORTANT !!

The latest release of Wukong CLI contains several breaking changes. Users must upgrade to the latest version, which is at least `0.0.4-beta1` in order to continue using the Wukong CLI. If you are using Wukong version < `0.0.4-beta1`, please follow the below instructions:

```
# Check the current version of Wukong CLI.  
$ wukong --version

# If your version is < 0.0.4-beta1.  
$ brew update  
$ brew upgrade wukong  

# Confirm that you're on version >= 0.0.4-beta1.  
$ wukong --version
wukong 0.0.4-beta1

# Delete the current config.  
$ rm ~/.config/wukong/config.toml

# Re-init the Wukong CLI.  
$ wukong init
```

```